benches/function.rs

```diff
@@ -51,7 +53,7 @@ fn main() {
   {
     let func = v8::Function::new(
       scope,
-      |_: &mut v8::HandleScope,
+      |_: &mut v8::PinScope,
```

Comment:
Are there other types of scopes like `PinCallbackScope`?

Reply:
There are pinscope and pincallbackscope. the other ones there aren't aliases for, they're super uncommon